pkg/ecdsa/dkg/member.go

```diff
@@ -111,24 +126,199 @@ func (skgm *symmetricKeyGeneratingMember) MarkInactiveMembers(
 	filter.FlushInactiveMembers()
 }
 
+// initializeTssRoundOne returns a member to perform next protocol operations.
+func (skgm *symmetricKeyGeneratingMember) initializeTssRoundOne() *tssRoundOneMember {
+	tssPartyID, groupTssPartiesIDs := generateTssPartiesIDs(
+		skgm.id,
+		skgm.group.MemberIDs(),
+	)
+
+	tssParameters := tss.NewParameters(
+		tss.EC(),
```

Comment:
Maybe we should call `S256()` to ask for secp256k1 explicitly in case `tss-lib` ever changes the default curve?

Reply:
See https://github.com/keep-network/keep-core/pull/3112/commits/c7c413d9e048f82d5f379eb0ce6821f60ac9bfa2